melange.opam.template

```diff
@@ -1,9 +1 @@
-pin-depends: [
-  [ "melange-compiler-libs.dev" "git+https://github.com/melange-re/melange-compiler-libs.git"]
-]
-
-install: [
-  [ "dune" "install" "-p" "%{name}%" ]
-]
-
-setenv: [ MELANGELIB = "%{_:lib}%/melange" ]
+setenv: [ MELANGELIB = "%{_:lib}%/melange/melange" ]
```

Comment:
I think this change introduces a regression as the final path becomes something like `/path/to/_opam/lib/melange/melange/melange`, which is non existent.

I will confirm (and potentially fix if needed) in #369.